applications/plugins/SofaAssimp/examples/AssimpObjLoader.scn

```diff
@@ -1,14 +1,13 @@
 <?xml version="1.0"?>
-<!-- For more details see: https://wiki.sofa-framework.org/tdev/wiki/Notes/NewLoaderArchitecture -->
 <Node name="Root" gravity="0 -9.81 0" dt="0.02">
     <RequiredPlugin name="SofaOpenglVisual"/>
     <RequiredPlugin name="Assimp" pluginName="SofaAssimp"/>
     <VisualStyle displayFlags="" />
     <DefaultPipeline verbose="0" />
     <BruteForceDetection name="N2" />
     <DefaultContactManager response="default" />
-    <MinProximityIntersection name="Proximity" alarmDistance="0.08" contactDistance="0.05" useSurfaceNormals="false"/>
-    <DefaultCollisionGroupManager />
+    <MinProximityIntersection name="Proximity" alarmDistance="0.08" contactDistance="0.05"/>
```

Comment:
just a copy past I think to have the same default component. In fact those component could even be removed because the scene are only loading the mesh without any simulation behind